dataactbroker/handlers/accountHandler.py

```diff
@@ -21,7 +23,11 @@
 from dataactcore.utils.statusCode import StatusCode
 from dataactcore.interfaces.function_bag import get_email_template, check_correct_password, updateLastLogin
 from dataactcore.config import CONFIG_BROKER
-from dataactcore.models.lookups import USER_STATUS_DICT, PERMISSION_TYPE_DICT, PERMISSION_MAP
+from dataactcore.models.lookups import (
+    PERMISSION_SHORT_DICT, PERMISSION_TYPE_DICT, USER_STATUS_DICT)
```

Comment:
The parens just allow us to use multiple lines. Pep8 has a 79-char line length, so I tend to use that when writing new code.